### ISSUE TITLE
[ERROR] : No module named idna

### ISSUE THREAD

**Issue by TheVakman** [sheagcraig/JSSImporter#96]
When I install the JSS Importer integration piece for AutoPkgr, I get an this error but it does install:
```
A Python exception occurred during the execution of autopkg, see the system log for more details.

[ERROR] : No module named idna
```

It does seem to install though.

This happens on Mac OS X 10.11 and macOS 10.12.
AutoPkg: 1.0.0
Git: 2.9.3
AutoPkgr: 1.4.2

**Comment by homebysix** [comment]
This seems like a duplicate of #95.

**Comment by TheVakman** [comment]
@homebysix Agreed. I'd close #95 though as it has a typo but relate it as the error is clear here.

**Comment by homebysix** [comment]
A source familiar with the matter says that this command may need to be run on fresh Sierra installs before JSSImporter and python-jss will work:

```
easy_install pip && pip install -I --user pyopenssl
```

Note: Don't put `sudo` in front of this command.

**Comment by macmule** [comment]
FWIW, just saw this on 10.10.5.. so might not be a 10.12 change but maybe an JSSImporter/AutoPKG change? 

(10.10.5 is a temp measure on this host.. promise!)

**Comment by sheagcraig** [comment]
Correct; it is not a Sierra issue. It is a macOS system python issue.

**Comment by gerardweese** [comment]
I ran into this issue recently and it was due to the version of pip that is installed on OSX via `easy_install`

Check your version with `pip --version`

If your not at least 9.0 run 
`sudo pip install --upgrade pip`
then re-run 
`sudo pip install -I --user pyopenssl`
